src/lib/statsv2.svelte

```diff
@@ -0,0 +1,179 @@
+<!--
+  statsv2.svelte:
+    Stats highlighting the value of fixNation's work-v2
+-->
+<script lang="ts">
+  import { base } from '$app/paths';
+  import { onMount, onDestroy } from 'svelte';
+  let sections = [
+    {
+      number: '1',
+      subtitle: 'largest clinic for homeless cats',
+      picture: 'statsv2/statImage1.png'
+    },
+    {
+      number: '115k',
+      subtitle: 'total cats spayed or neutered',
+      picture: 'statsv2/statImage2.png'
+    },
+    {
+      number: '12k',
+      subtitle: 'TNR volunteers trained',
+      picture: 'statsv2/statImage3.png'
+    },
+    {
+      number: '75%',
+      subtitle: 'cats are feral',
+      picture: 'statsv2/statImage4.png'
+    },
+    {
+      number: '89%',
+      subtitle: 'profits go directly to FixNation programs',
+      picture: 'statsv2/statImage5.png'
+    },
+    {
+      number: '21%',
+      subtitle: 'of cats abandoned because their of their residency',
+      picture: 'statsv2/statImage6.png'
+    }
+  ];
+
+  let screenWidth: number;
+
+  const updateScreenWidth = () => {
+    screenWidth = window.innerWidth;
+  };
+
+  onMount(() => {
+    if (typeof window !== 'undefined') {
+      window.addEventListener('resize', updateScreenWidth);
+      updateScreenWidth();
+    }
+  });
+
+  onDestroy(() => {
+    if (typeof window !== 'undefined') {
+      window.removeEventListener('resize', updateScreenWidth);
+    }
+  });
+</script>
+
+<h1>
+  FixNation’s mission is to reduce the population of homeless cats by sterilizing as many as
+  possible while demonstrating the effectiveness of Trap-Neuter-Return and colony management for the
+  humane care of homeless cats.
+</h1>
+<div>
+  {#if screenWidth > 675}
+    <div class="slider">
+      {#each sections as section}
+        <section class="section">
+          <img src={section.picture} />
```

Comment:
Can you add an `alt` tag to each image for accessibility? An `alt` tag shows some text while the image itself loads, which helps with screen reading and slow connections. I'd suggest adding an "alt" property to each section and using them like this:

```suggestion
          <img src={section.picture} alt={section.alt} />
```